### PR TITLE
Random quote generator added

### PR DESCRIPTION
**Pull request by JamBai2812**
Hard coded an array of quotes from website. 
Generated three distinct random numbers to pick random quotes form the array.
Added quotes to quote cards.

### CODE REVIEW

File: src/Pages/InspirationalQuotesPage/InspirationalQuotesPage.tsx [hunk 1]
```diff
@@ -2,23 +2,106 @@
 import {Quotecard} from "../../Components/Quotecard/Quotecard";
 import "./InspirationalQuotesPage.scss"
 
-export function InspirationalQuotesPage(){
-    return(
+export function InspirationalQuotesPage() {
+
+    const quotes = [
```
Comment (MichaelWalker) [inline src/Pages/InspirationalQuotesPage/InspirationalQuotesPage.tsx]: Nice.

To make this even nicer we can do 2 things

1) move it outside of the `function` (at the moment we are re-declaring this array every time we re-render, which slightly wasteful (although computers are fast, so we probably get away with it)

2) give it a type! We can make an interface for these quotes, and then the type system can help us out! :)

File: src/Pages/InspirationalQuotesPage/InspirationalQuotesPage.tsx [hunk 2]
```diff
@@ -2,23 +2,106 @@
...
+                        quote={quotes[quoteTwo].quote}
+                        author={quotes[quoteTwo].name}
+                    />
+                </li>
+                <li>
+                    <Quotecard
+                        quote={quotes[quoteThree].quote}
+                        author={quotes[quoteThree].name}
+                    />
+                </li>
```
Comment (MichaelWalker) [inline src/Pages/InspirationalQuotesPage/InspirationalQuotesPage.tsx]: can simplify this slightly with 
```jsx
{numbers
    .map(number => quotes[number])
    .map(quote => <li><Quotecard quote={quote.quote} author={quote.author}/></li>)
}
```

File: src/Pages/InspirationalQuotesPage/InspirationalQuotesPage.tsx [hunk 3]
```diff
@@ -2,23 +2,106 @@
...
+        {
+            name: "Viktor Alexandrov",
+            quote: "“Gravity hurts.”"
+        }
+    ];
+
+    var numbers = [];
+    while(numbers.length < 3){
+        var num = Math.floor(Math.random() * (quotes.length));
+        if(numbers.indexOf(num) === -1) numbers.push(num);
```
Comment (MichaelWalker) [inline src/Pages/InspirationalQuotesPage/InspirationalQuotesPage.tsx]: this works fine, but we can make it a little more readable with
```js
if (!numbers.includes(num)) {
    numbers.push(num);
}
```

**Review by MichaelWalker — CHANGES_REQUESTED** [review]

File: src/Pages/InspirationalQuotesPage/InspirationalQuotesPage.tsx (hunk 1 above)
Comment (JamBai2812) [reply src/Pages/InspirationalQuotesPage/InspirationalQuotesPage.tsx]: Thanks, Mike. 
Changes made.

**Comment by JamBai2812** [comment]
changes made.

File: src/Pages/InspirationalQuotesPage/InspirationalQuotesPage.tsx [hunk 4]
```diff
@@ -2,23 +2,103 @@
...
+    while (numbers.length < 3) {
+        var num = Math.floor(Math.random() * (quotes.length));
+        if (!numbers.includes(num)) {
+            numbers.push(num)
+        }
+    }
+    
+    const quoteOne: number = numbers[0];
+    const quoteTwo: number = numbers[1];
+    const quoteThree: number = numbers[2];
```
Comment (MichaelWalker) [inline src/Pages/InspirationalQuotesPage/InspirationalQuotesPage.tsx:92]: We don't need these any more either! :)

**Review by MichaelWalker — APPROVED** [review]
Could delete some unused things, but looks good!

**Pull request merged**